files/ja/glossary/site/index.md

```diff
@@ -19,7 +19,7 @@ l10n:
 
 これらは、登録可能なドメインが `mozilla.org` であることから、同じサイトであると考えられます。
 
-- `https://developer.mozilla.org/en-US/docs/`
+- `https://developer.mozilla.org/ja/docs/`
```

Comment:
This is an example of URL, so it doesn't have to be modifed to /ja/.

Reply:
The other locales use their own locale codes for the examples, so I think that it makes sense for Japanese to do the same!